packages/@icon-magic/cli/src/index.ts

```diff
@@ -18,7 +18,11 @@ program
   .description(
     'construct flavors of an icon from its variants, after applying the build plugins.'
   )
-  .action(async inputPaths => {
+  .option(
+    '-h, --hashing',
+    'If false, Icon Magic will run build and generate on all icons including those that have not changed since the previous execution. If true it will only run on icons that have changed assets. Defaults to true.'
```

Comment:
Make it less verbose. 

"When true(default), builds only those icon variants that have changed since the previous execution. If false, builds all icons. "